ios/FirebaseContinue/Example/Podfile.lock

```diff
@@ -0,0 +1,45 @@
+PODS:
```

Comment:
Podfile.lock should be in .gitignore and not checked. (They may be checked in for apps, but should not be for libraries)